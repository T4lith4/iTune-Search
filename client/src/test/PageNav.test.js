import React from "react";
import PageNav from "../components/page-nav";
import renderer from "react-test-renderer";

test("renders correctly", () => {
  const tree = renderer.create(<PageNav />).toJSON();

  expect(tree).toMatchSnapshot();
});