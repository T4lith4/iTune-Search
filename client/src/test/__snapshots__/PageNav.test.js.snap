// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="nav-link-container"
>
  <a
    className="nav-link hide 
          "
    href="/"
  >
    ←
  </a>
  <a
    className="nav-link hide
          "
    href="/"
  >
    →
  </a>
</div>
`;
